.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,20 @@
+name: golangci-lint
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+      - main
+  pull_request:
+permissions:
+  contents: read
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v2
+      - uses: actions/checkout@v2
+      - name: golangci-lint
```

Comment:
just curious if action this actually calls `gofmt` too ?
According to https://golangci-lint.run/usage/linters/ only minimal linters are enabled by default

Reply:
annoyingly it doesn't ; need to enable some more indeed

Reply:
fwiw gofmt lint is enabled now